amundsen_application/static/js/utils/dateUtils.ts

```diff
@@ -0,0 +1,9 @@
+import * as moment from 'moment-timezone';
+
+export const DATE_FORMAT = 'MMM DD, YYYY';
+export const DATE_FORMAT_LONG = 'MMMM Do YYYY [at] h:mm:ss a';
```

Comment:
nit: I'd lean towards event more descriptive names because eventually we might support more formats and then it gets less and less intuitive to understand what format is which. For this one I might go with `DATE_TIME_FORMAT_LONG` since it will include the time.